nixos/tests/all-tests.nix

```diff
@@ -71,6 +71,7 @@ in
   convos = handleTest ./convos.nix {};
   corerad = handleTest ./corerad.nix {};
   couchdb = handleTest ./couchdb.nix {};
+  cri-o = handleTest ./cri-o.nix {};
```

Comment:
All of the other OCI container related tests are `x86_64` only, may as well do the same for this one.

```suggestion
  cri-o = handleTestOn ["x86_64-linux"] ./cri-o.nix {};
```

Reply:
Fixed, thanks! :pray: